data/human/free worlds 4 epilogue.txt

```diff
@@ -32,7 +32,7 @@ mission "FW Epilogue: Ijs and Katya"
 					goto excitement
 				`	"It's kind of hard to adjust to civilian life."`
 					goto adjust
-				`	"I want to explore the rest of the galaxy, but apparently that part of the game hasn't been implemented yet."`
+				`	"Is this it? Is the game over now that I completed the free worlds campaign?"`
```

Comment:
I am not a fan of the fourth wall breaks referring to "missions" and "campaigns" ... while technically true, it feels sort of cheesy and trivializes the whole story aspect, and just feels _wrong_. Also, how do you even define the end of FW? Are the epilogue campaigns part of it or not? I don't think a campaign should ever have a hard-defined end point.
```suggestion
				`	"Is this it? Is the game over now that humanity's at peace again?"`
```